bundles/org.openhab.binding.yioremote/src/main/java/org/openhab/binding/yioremote/internal/YIOremoteDockHandler.java

```diff
@@ -0,0 +1,219 @@
+                                                        updateChannelString(GROUP_OUTPUT, YIODOCKSTATUS,
+                                                                yioremoteDockwebSocketClient.getstringreceivedstatus());
+                                                    } else {
+                                                        yioremotedockactualstatus = YIOREMOTEDOCKHANDLESTATUS.CONNECTIONFAILED;
+                                                        updateStatus(ThingStatus.OFFLINE,
+                                                                ThingStatusDetail.CONFIGURATION_ERROR,
+                                                                "Connection lost no ping from YIO DOCK");
+                                                        updateState(GROUP_OUTPUT, YIODOCKSTATUS, UnDefType.UNDEF);
+                                                        pollingJob.cancel(true);
+                                                    }
+                                                } else {
+                                                    yioremotedockactualstatus = YIOREMOTEDOCKHANDLESTATUS.CONNECTIONFAILED;
+                                                    updateStatus(ThingStatus.OFFLINE,
+                                                            ThingStatusDetail.CONFIGURATION_ERROR,
+                                                            "Connection lost no ping from YIO DOCK");
+                                                    updateState(GROUP_OUTPUT, YIODOCKSTATUS, UnDefType.UNDEF);
+                                                    pollingJob.cancel(true);
+                                                }
+                                            } catch (Exception e) {
+                                                logger.warn("Error during initializing the WebSocket polling Thread {}",
+                                                        e.getMessage());
+                                            }
+                                        }
+                                    };
+
+                                    pollingJob = scheduler.scheduleWithFixedDelay(heartbeatpolling, 0, 30,
+                                            TimeUnit.SECONDS);
+                                } catch (Exception e) {
+                                    updateChannelString(GROUP_OUTPUT, YIODOCKSTATUS, "Connection/Configuration Error");
+                                }
+                            } else {
+                                logger.debug("authentication to YIO dock not ok");
+                                yioremotedockactualstatus = YIOREMOTEDOCKHANDLESTATUS.AUTHENTICATEDFAILED;
+                                updateStatus(ThingStatus.OFFLINE);
+                            }
+                        } else {
+                            logger.debug("authentication error YIO dock");
```

Comment:
Instead of logging it, you could set the Thing state to OFFLINE.

Reply:
resolved